### PR TITLE
Sanity check script

### PR DESCRIPTION
**Pull request by justinemarie**
The idea would be to run something like this in Travis so we don't have any more edits that completely bork modules and configurations end to end.

### CODE REVIEW

**Comment by justinemarie** [comment]
Wait, where are these other commits from y'all coming from? I thought I hadn't pulled from anywhere other than develop...

**Comment by changlan** [comment]
If the script is for Travis only, would it be better to put that inside .travis directory?

File: sanity_check.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+BESSCTL='./bessctl/bessctl'
+SCRIPTS='./bessctl/conf/samples'
+OUTFILE='sanity_check.out'
+
+TESTS='exactmatch.bess
```
Comment (melvinw) [inline sanity_check.sh]: why not run everything in bessctl/conf/samples that doesn't require bringing up physical ports?

Comment (melvinw) [inline sanity_check.sh]: (if these are the those scripts ignore me)

**Comment by melvinw** [comment]
Merged manually. Thanks! 
pup

**Comment by sangjinhan** [comment]
No dawgs!
giphy

**Comment by justinemarie** [comment]
How do we go about integrating this into Travis?

**Pull request closed without merging**